### PR TITLE
adding python language

### PR DESCRIPTION
**Pull request by AaquibAli**
**Description**

Lang added : python

This PR fixes # NA

**Notes for Reviewers:**

Ready to review!

**[Signed commits](https://github.com/DevsTrons/hello-world/blob/master/CONTRIBUTING.md)**

- [x] Yes, I am a Member of **DEVSTRONS'**

**Discord id** (eg: DEVSTRONS'#0238):  aaquib ali#3729

### CODE REVIEW

File: lang/python.md [hunk 1]
```diff
@@ -2,10 +2,154 @@
...
+# Popular Python editor 
+- [Visual Studio Code](https://code.visualstudio.com/download) by Microsoft
+
+- [Sublime](https://www.sublimetext.com/) Text by Jon Skinner
+
+- [Atom](https://atom.io/) by GitHub
+
+- [PyCharm](https://www.jetbrains.com/pycharm/) by JetBrains
+
+# Some Cool projects in Java
```
Comment (ghulamyazdani) [inline lang/python.md:135]: Java??
Need changes here.
Change it to "Some Cool Projects In Python"

**Review by ghulamyazdani — CHANGES_REQUESTED** [review]

File: lang/python.md [hunk 2]
```diff
@@ -2,10 +2,154 @@
...
+
+## The Best Place to Start :
+efore you can do anything with a programming language, you first need to configure the development environment. Now, we’re going to cover how to setup Python and the interpreter that will execute your custom programs.
+
+Python comes pre-installed on Mac and a majority of Linux distributions. However, you may need to download an updated version depending on how old your system is.
+
+**TIP:** You can easily check your Python version by opening the terminal and running the following command:
+
+```
+    Python -v
```
Comment (amino19) [inline lang/python.md:108]:
```suggestion
    python --version
```

File: lang/python.md [hunk 3]
```diff
@@ -2,10 +2,154 @@
...
+
+**Mac:** Most Mac OS X computers already have Python 2.x installed, which is perfectly fine to use. The best way to install the latest version is to use [Homebrew](https://brew.sh/) to activate and manage them. You’ll find instructions on [how to do that here](https://gist.github.com/mjhea0/3c7d97e1d20ab86eb41e). You can also download Python directly from the PSF if you prefer.
+
+**Linux:** Python is included with most distributions of Linux. Check your current version and be sure to upgrade using the package manager, if necessary.
+
+**Windows:** Just download Python from the PSF.
+
+**TIP:** If you’re using Windows, be sure to select the option that adds Python.exe to your system path during installation. You can do this by selecting the option next to “Add python.exe to path,” and then choosing your local hard drive as the install location.
+
+## Which Version Should I Use?
```
Comment (amino19) [inline lang/python.md:123]:
```suggestion
**Which Version Should I Use?**
```

File: lang/python.md [hunk 4]
```diff
@@ -2,10 +2,154 @@
...
+Python comes pre-installed on Mac and a majority of Linux distributions. However, you may need to download an updated version depending on how old your system is.
+
+**TIP:** You can easily check your Python version by opening the terminal and running the following command:
+
+```
+    Python -v
+```
+If you’re running Windows, you’ll need to download Python from the Python Software Foundation (link below).
+
+## How Do I Get Python?
```
Comment (amino19) [inline lang/python.md:112]: This & the below section must be included under, `## Setting up Python development environment`, include this at last i.e, L84 & remove this line.

```suggestion

```

File: lang/python.md [hunk 5]
```diff
@@ -2,10 +2,154 @@
...
+**Linux:** Python is included with most distributions of Linux. Check your current version and be sure to upgrade using the package manager, if necessary.
+
+**Windows:** Just download Python from the PSF.
+
+**TIP:** If you’re using Windows, be sure to select the option that adds Python.exe to your system path during installation. You can do this by selecting the option next to “Add python.exe to path,” and then choosing your local hard drive as the install location.
+
+## Which Version Should I Use?
+Long story short, it’s entirely up to you which version of Python you use! There’s no right or wrong answer, and it’s painless to make the jump from one version to another should you realize you need to do so later on.
+
+# Popular Python editor
```
Comment (amino19) [inline lang/python.md:126]:
```suggestion
## Popular Python IDEs/Editors 
```

File: lang/python.md [hunk 6]
```diff
@@ -2,10 +2,154 @@
...
+# Popular Python editor 
+- [Visual Studio Code](https://code.visualstudio.com/download) by Microsoft
+
+- [Sublime](https://www.sublimetext.com/) Text by Jon Skinner
+
+- [Atom](https://atom.io/) by GitHub
+
+- [PyCharm](https://www.jetbrains.com/pycharm/) by JetBrains
+
+# Some Cool projects in Python
```
Comment (amino19) [inline lang/python.md:135]:
```suggestion
## Some cool projects in Python
```

File: lang/python.md [hunk 7]
```diff
@@ -2,10 +2,154 @@
...
+
+- [System Design Primer](https://github.com/donnemartin/system-design-primer) : This repo is an organized collection of resources to help you learn how to build systems at scale.
+
+- [Face Recognition](https://github.com/ageitgey/face_recognition) : Recognize and manipulate faces from Python or from the command line with the world’s simplest face recognition library.
+
+- [snallygaster](https://github.com/hannob/snallygaster) : Tool to scan for secret files on HTTP servers.
+
+- [Cookiecutter](https://github.com/cookiecutter/cookiecutter) : A command-line utility that creates projects from cookiecutters (project templates)
+ 
+# Conclusion
```
Comment (amino19) [inline lang/python.md:150]:
```suggestion
## Conclusion
```

**Review by amino19 — CHANGES_REQUESTED** [review]
@AaquibAli: Need to change a few stuffs, & then we will be good to go! :rocket:

File: lang/python.md [hunk 8]
```diff
@@ -2,10 +2,154 @@
...
+
+- **First-choice Language**
+
+- **The Flexibility of Python Language**
+
+- **Use of python in academics**
+
+- **Automation** 
+
+# Why learn Python?
```
[the remark this answers is not in the document]
Comment (amino19) [reply lang/python.md]: @AaquibAli: Still not resolved!

File: lang/python.md [hunk 9]
```diff
@@ -2,10 +2,155 @@
...
+
+**Mac:** Most Mac OS X computers already have Python 2.x installed, which is perfectly fine to use. The best way to install the latest version is to use [Homebrew](https://brew.sh/) to activate and manage them. You’ll find instructions on [how to do that here](https://gist.github.com/mjhea0/3c7d97e1d20ab86eb41e). You can also download Python directly from the PSF if you prefer.
+
+**Linux:** Python is included with most distributions of Linux. Check your current version and be sure to upgrade using the package manager, if necessary.
+
+**Windows:** Just download Python from the PSF.
+
+**TIP:** If you’re using Windows, be sure to select the option that adds Python.exe to your system path during installation. You can do this by selecting the option next to “Add python.exe to path,” and then choosing your local hard drive as the install location.
+
+**Which Version Should I Use?**
```
Comment (amino19) [inline lang/python.md:124]: @AaquibAli: Need one empty line after this, just use `Enter`, so it will be fixed!
Same as the previous one! : )

**Review by amino19 — CHANGES_REQUESTED** [review]

File: lang/python.md [hunk 10]
```diff
@@ -2,10 +2,157 @@
 lang: Python
 syntax: python
 title: Python is a snake language!
+author : AaquibAli
 ---
 
 ```python
 print("hello world")
 ```
+ ## What is Python?
```
Comment (Panquesito7) [inline lang/python.md:11]: There's an extra space here which will make the header not work well.
```suggestion
## What is Python?
```

File: lang/python.md [hunk 11]
```diff
@@ -2,10 +2,155 @@
...
+One of the most important things you’ll do when working with any programming language is setup a development environment which allows you to execute the code you write. Without this, you will never be able to check your work and see if your website or application is free of syntax errors.
+
+With Python, you also need something called an interpreter that converts your code – which makes up the entirety of your application – to something the computer can read and execute. Without this interpreter, you’ll have no way to run your code.
+
+To convert your code, you must first use a Python shell, which calls upon the interpreter through something called a “bang” line.
+
+As for creating an application or file, there are two ways to do this. You can create a program using a simple text editor like WordPad, or Notepad++. You can also create a program using a Python shell. There are advantages and disadvantages to each method, which we’ll discuss next.
+## How Do I Get Python? 
+
+**Python Shell vs Text File:**
```
[the remark this answers is not in the document]
Comment (Panquesito7) [reply lang/python.md:93]: This is not needed when using bold text. It is only needed when using headers. 🙂

[the remark this answers is not in the document]
Comment (amino19) [reply lang/python.md:93]: Actually, we're using Bold keywords for a header for now!  : )

File: lang/python.md (hunk 10 above)
Comment (amino19) [reply lang/python.md:12]: @AaquibAli: As Pan requested, you've to change this as well!
I forgot this changes to let you made in voice chat! XD
My bad!

File: lang/python.md [hunk 12]
```diff
@@ -2,10 +2,168 @@
...
+
+- **It could lead to a well-paid career :** Data suggests that the median annual salary for those with Python skills is [around £65,000 in the UK](https://www.itjobswatch.co.uk/jobs/uk/python.do) 
+
+- **There will be many job opportunities :** Given that Python can be used in many emerging technologies, such as AI, machine learning, and data analytics, it’s likely that it’s a future-proof skill. Learning Python now could benefit you across your career. 
+
+### What is Python used for?
+
+Clearly, Python is a popular and in-demand skill to learn. But what is python programming used for? We’ve already briefly touched on some of the areas it can be applied to, and we’ve expanded on these and more Python examples below. 
+
+**Python can be used for :**
```
Comment (amino19) [inline lang/python.md:60]:
```suggestion
#### Python can be used for:

```

File: lang/python.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (amino19) [reply lang/python.md:93]:
```suggestion
#### Python Shell vs Text File:

```

File: lang/python.md (hunk 9 above)
Comment (amino19) [reply lang/python.md:133]:
```suggestion
#### Which Version Should I Use?
```

**Review by amino19 — CHANGES_REQUESTED** [review]
@AaquibAli: The last commit, have to make some small changes! :slightly_smiling_face: 

Also, try to add some emojis as we've [here in Java](https://github.com/devstrons/hello-world/blob/main/lang/java.md).

File: lang/python.md [hunk 13]
```diff
@@ -2,10 +2,154 @@
...
+
+- **Language development**
+
+- **Finance**
+
+- **SEO**
+
+- **Design**
+
+# Setting up a Python development environment
```
[the remark this answers is not in the document]
Comment (amino19) [reply lang/python.md]: @AaquibAli: Still not resolved, man!

File: lang/python.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (Panquesito7) [reply lang/python.md:93]: `####` won't make it look bold. We would need to use `#### **Text here**` to make it a header and make it look bold. 🙂

File: lang/python.md [hunk 14]
```diff
@@ -9,15 +9,15 @@ author : AaquibAli
...
 
 Python is considered a scripting language, like [Ruby](https://techterms.com/definition/ruby) or [Perl](https://techterms.com/definition/perl) and is often used for creating Web [applications](https://techterms.com/definition/application) and dynamic Web content. It is also supported by a number of 2D and 3D imaging programs, enabling users to create custom [plug-ins](https://techterms.com/definition/plugin) and extensions with Python. Examples of applications that support a Python API include GIMP, Inkscape, Blender, and Autodesk Maya.
 
 Scripts written in Python ([.PY](https://fileinfo.com/extension/py) files) can be [parsed](https://techterms.com/definition/parse) and run immediately. They can also be saved as a compiled programs ([.PYC](https://fileinfo.com/extension/pyc) files), which are often used as programming modules that can be referenced by other Python programs.
 
-### Why is Python so popular?
+### 🤔 Why is Python so popular?
```
Comment (amino19) [inline lang/python.md:20]:
```suggestion
### 📚 Why is Python so popular?
```

File: lang/python.md [hunk 15]
```diff
@@ -53,11 +53,12 @@ We got why `Python is so popular` at the moment above, but why should one learn
 
 - **There will be many job opportunities :** Given that Python can be used in many emerging technologies, such as AI, machine learning, and data analytics, it’s likely that it’s a future-proof skill. Learning Python now could benefit you across your career. 
 
-### What is Python used for?
+### 	🧐 What is Python used for?
```
Comment (amino19) [inline lang/python.md:56]:
```suggestion
### 🧐 What is Python used for?
```

File: lang/python.md (hunk 12 above)
Comment (amino19) [reply lang/python.md:60]: @AaquibAli: Still not done!

File: lang/python.md [hunk 16]
```diff
@@ -88,9 +89,9 @@ To convert your code, you must first use a Python shell, which calls upon the in
 
 As for creating an application or file, there are two ways to do this. You can create a program using a simple text editor like WordPad, or Notepad++. You can also create a program using a Python shell. There are advantages and disadvantages to each method, which we’ll discuss next.
 
-### How Do I Get Python? 
+### 👀 How Do I Get Python?
```
Comment (amino19) [inline lang/python.md:92]:
```suggestion
### ⚙️ How Do I Get Python? 
```

File: lang/python.md [hunk 17]
```diff
@@ -88,9 +89,9 @@ To convert your code, you must first use a Python shell, which calls upon the in
 
 As for creating an application or file, there are two ways to do this. You can create a program using a simple text editor like WordPad, or Notepad++. You can also create a program using a Python shell. There are advantages and disadvantages to each method, which we’ll discuss next.
 
-### How Do I Get Python? 
+### 👀 How Do I Get Python? 
 
-**Python Shell vs Text File:**
+**Python Shell vs Text File :**
```
Comment (amino19) [inline lang/python.md:94]:
```suggestion
#### Python Shell vs Text File:
```

File: lang/python.md [hunk 18]
```diff
@@ -106,7 +107,7 @@ This makes it more difficult to spot errors in your code, and it can also be fru
 
 There are text editors with increased functionality – like Notepad++ – which were specifically developed with programming in mind.
 
-### How to Get Started with Python?
+### 👻 How to Get Started with Python?
```
Comment (amino19) [inline lang/python.md:110]:
```suggestion
### ⚙️ How to Get Started with Python?
```

File: lang/python.md [hunk 19]
```diff
@@ -130,11 +131,11 @@ If you need to download Python, the best place to go is the [official site](http
 
 **TIP:** If you’re using Windows, be sure to select the option that adds Python.exe to your system path during installation. You can do this by selecting the option next to “Add python.exe to path,” and then choosing your local hard drive as the install location.
 
-**Which Version Should I Use?**
+**⚙️ Which Version Should I Use?**
 
-Long story short, it’s entirely up to you which version of Python you use! There’s no right or wrong answer, and it’s painless to make the jump from one version to another should you realize you need to do so later on.
+   Long story short, it’s entirely up to you which version of   Python you use! There’s no right or wrong answer, and it’s painless to make the jump from one version to another should you realize you need to do so later on.
```
Comment (amino19) [inline lang/python.md:136]:
```suggestion
Long story short, it’s entirely up to you which version of   Python you use! There’s no right or wrong answer, and it’s painless to make the jump from one version to another should you realize you need to do so later on.
```

File: lang/python.md [hunk 20]
```diff
@@ -160,10 +161,10 @@ Long story short, it’s entirely up to you which version of Python you use! The
...
  
-## Conclusion
+## ✨ Conclusion 
 
 That concludes our look at what Python programming can be used for. As you can see, there are many applications for this popular language, with a wide support network and a diverse range of libraries that can help. 
 
 There are many reasons why you might want to start learning Python. It’s a future-proof and in-demand skill that’s required across all kinds of industries. What’s more, we have a broad selection of Python courses that can help you either master the basics or develop some more specific skills.
 
-## Thanks for Reading :)
+## 🤗 Thanks for Reading :)
```
Comment (amino19) [inline lang/python.md:170]:
```suggestion
## Thanks for Reading 🤗 
```

**Review by amino19 — CHANGES_REQUESTED** [review]
@AaquibAli: Go through all comments, I made previously, also need to replace some emojis..
then, we're good to go! :rocket:

**Comment by amino19** [comment]
@AaquibAli: Great work! LGTM! :partying_face:

**Comment by CuriousCoderss** [comment]
> @AaquibAli: Great work! LGTM! 🥳

Thanks 🥰 .Not without your support 😉

File: lang/python.md [hunk 21]
```diff
@@ -2,10 +2,169 @@
...
+
+- **It could lead to a well-paid career :** Data suggests that the median annual salary for those with Python skills is [around £65,000 in the UK](https://www.itjobswatch.co.uk/jobs/uk/python.do) _(at present)_.
+
+- **There will be many job opportunities :** Given that Python can be used in many emerging technologies, such as AI, machine learning, and data analytics, it’s likely that it’s a future-proof skill. Learning Python now could benefit you across your career. 
+
+### 🧐 What is Python used for?
+
+Clearly, Python is a popular and in-demand skill to learn. But what is python programming used for? We’ve already briefly touched on some of the areas it can be applied to, and we’ve expanded on these and more Python examples below. 
+
+#### Python can be used for:
```
Comment (Panquesito7) [inline lang/python.md:60]: I think the `:` should be removed here.
```suggestion
#### Python can be used for
```
